<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageMhwd</class>
 <widget class="QWidget" name="PageMhwd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="buttonInstallFree">
       <property name="text">
        <string>&lt;a href=&quot;#&quot;&gt;Auto Install Open-Source Driver&lt;/a&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="buttonInstallNonFree">
       <property name="text">
        <string>&lt;a href=&quot;#&quot;&gt;Auto Install Proprietary Driver&lt;/a&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>2</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="styleSheet">
        <string notr="true">QPushButton color rgb(0, 0, 255)</string>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Driver</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Open-source</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Installed</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxShowAll">
       <property name="text">
        <string>Show all devices</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="installAction">
   <property name="text">
    <string>Install</string>
   </property>
  </action>
  <action name="removeAction">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="reinstallAction">
   <property name="text">
    <string>Reinstall</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
